pkg/sbproxy/reconcile/resyncer.go

```diff
@@ -61,7 +61,7 @@ func (r *resyncJob) process(ctx context.Context) {
 		return
 	}
 
-	smBrokers, err := r.getBrokersFromSM(ctx)
+	smBrokers, err := r.getBrokersFromSM(ctx, r.options.BrokerBlacklist)
```

Comment:
r.Options is part of r and r.getBrokersFromSM is part of r. Also the blacklist is static for the whole existence of r. So you don't need to pass it as parameter

Reply:
Addressed